spec/models/external_work_spec.rb

```diff
@@ -30,13 +30,32 @@
     URLS = ["http://the--ivorytower.livejournal.com/153798.html"]
 
     URLS.each do |url|
-      let(:valid_url) {build(:external_work, url: url)}
+      # Test each of the possible valid response codes
+      WebMock.stub_request(:any, url).to_return({ status: 200, body: "Success" }, { status: 301, body: "Moved Permanently" }, { status: 302, body: "Found" }, { status: 307, body: "Temporary Redirect" }, { status: 308, body: "Permanent Redirect" })
```

Comment:
This line doesn't seem to work the way it's intended to. If I check out this commit but revert the changes to `app/validators/url_active_validator.rb`, it should fail on the 307/308 tests. Instead, all five tests pass.

This line of code isn't in a `let`, a `before`, or inside a test. That means it gets run once when the file is first loaded. And I think WebMock cleans up any mocked requests after each test, so this test is actually performing an external request each time. (With the possible exception of the very first test.)